mmv1/third_party/terraform/services/secretmanagerregional/data_source_secret_manager_regional_secret_version_test.go

```diff
@@ -201,3 +250,40 @@ func testAccCheckDataSourceSecretManagerRegionalRegionalSecretVersion(n, expecte
 		return nil
 	}
 }
+
+func testAccCheckDataSourceSecretManagerRegionalRegionalSecretVersionSecretData(datasource, resource string) resource.TestCheckFunc {
```

Comment:
```suggestion
func testAccCheckSecretManagerRegionalRegionalSecretVersionSecretDataDatasourceMatchesResource(datasource, resource string) resource.TestCheckFunc {
```

Reply:
I've updated the function name wherever required.